docs/managed-datahub/datahub-api/entity-events-api.md

```diff
@@ -378,13 +378,45 @@ This event is emitted when a description has been added to an entity on DataHub.
 }
 ```
 
+### Modify Description Event
+
+This event is emitted when an existing description of an entity has been modified on DataHub.
```

Comment:
Since we already have "Add Description Event" (from [this PR](https://github.com/datahub-project/datahub/pull/10237/files#diff-808deca731c65a066739bf2fcd250a88da12e1ecd6462942282205de67d49384R349)), can you make this a bit more clear that this is just for schema fields and not entities? Maybe something like "Modify Schema Description Event".

If that means combining these events that works as well.

Reply:
Reverted this addition to the docs to avoid confusion (this addition also isn't strictly necessary for this PR).

Reply:
Great! LGTM, let me know when you're ready for review